jobs/ajitava_cards_gold.py

```diff
@@ -0,0 +1,93 @@
+"""
+This script will fetch all the cards of a trello board that according to the below criteria:
+- Get the data for a defined data range
+Save the data to Delta Lake Gold table
+This script is scheduled to run twice a week
+"""
+
+import os
+import sys
+from datetime import datetime, timedelta
+import argparse
+from loguru import logger
+from pyspark.sql.functions import col
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+from helpers import spark_helper as sh
+from config import delta_table_conf as dc
+
+def get_cards(spark, start_date, end_date):
+    """
+    Fetch all the cards data of a user within the mentioned date range
+    :param spark: Spark session
+    :param start_date: Start date
+    :param end_date: End date
+    :return none
+    """
+    path = dc.ajitava_cards_gold_path
+    try:
+        # Fetch the cards data from the refined cards silver delta table
+        refined_cards_data = spark.read.format(
+            "delta").load(dc.cards_unique_silver_table_path)
+        logger.info(f'Fetched the refined cards data from silver table')
+        
+        #Fetch the data based on user
+        user_card_data = refined_cards_data.filter((col("card_members").like("Ajitava Deb")))
```

Comment:
you had used like("Ajitava Deb"). So, it only shows up the cards on which you alone worked. 

When I tried with like("%Ajitava%"), this showed up all other cards (which you had worked with others too) showed up. 
So you might want to update the query so that you don't miss out on those data.

Reply:
Done but using like("Ajitava Deb") showed up one data from Jan 2023 and using like("%Ajitava%") showed up all the data from Jan 2023 including the data fetched while using like("Ajitava Deb"). I am not sure why.